crates/telio-model/src/api_config.rs

```diff
@@ -176,6 +176,8 @@ pub struct FeatureDerp {
     pub tcp_keepalive: Option<u32>,
     /// Derp will send empty messages after this many seconds of not sending/receiving any data [default 60s]
     pub derp_keepalive: Option<u32>,
+    /// Enable polling of remote peer states to reduce derp traffic
+    pub enable_polling: Option<bool>,
```

Comment:
Is there a hard limit on what poll timer should be, or would it make sence to make it remote configurable ?

Reply:
It's already configurable, poll timer will use derp_keepalive interval as this polling is a replacement for derp keepalives